packages/user/Supervisor/ui/src/supervisor.ts

```diff
@@ -203,6 +197,20 @@ export class Supervisor implements AppInterface {
         return this.parentOrigination.app;
     }
 
+    getActiveQueryToken(): string {
```

Comment:
This can be private (and therefore moved above the constructor just for organization)